docker-compose-examples/keycloak-postgres.yml

```diff
@@ -6,20 +6,22 @@ volumes:
 
 services:
   postgres:
-      image: postgres
+      build: ./postgres
```

Comment:
Remove this change. We should not require users to build a Postgres image to use the example. I'd say the schema should be set to public here and it's up to the users to figure out how to add a schema to Postgres and not something we should cover on our end.